test/integration/targets/vmware_guest/tasks/clone_set_manual_mac_address.yml

```diff
@@ -0,0 +1,28 @@
+- name: clone VM from template and set manual mac address
+  vmware_guest:
+    validate_certs: False
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    name: test_vm1
+    template: "{{ virtual_machines[0].name }}"
+    datacenter: "{{ dc1 }}"
+    state: poweredoff
+    folder: "{{ virtual_machines[0].folder }}"
+    networks:
+      - name: VM Network
+        ip: 192.168.10.10
+        netmask: 255.255.255.0
+        gateway: 192.168.10.254
+        mac: aa:bb:cc:dd:aa:b0
+  register: clone_manual_mac_address
```

Comment:
Oh, I get the following error :-(

```
fatal: [testhost]: FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": [],
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "DC0",
            "datastore": null,
            "delete_from_inventory": false,
            "disk": [],
            "esxi_hostname": null,
            "folder": "F0",
            "force": false,
            "guest_id": null,
            "hardware": {},
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "test_vm1",
            "name_match": "first",
            "networks": [
                {
                    "gateway": "192.168.10.254",
                    "ip": "192.168.10.10",
                    "mac": "aa:bb:cc:dd:aa:b0",
                    "name": "VM Network",
                    "netmask": "255.255.255.0",
                    "type": "static"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "poweredoff",
            "state_change_timeout": 0,
            "template": "DC0_H0_VM0",
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_customization_timeout": 3600,
            "wait_for_ip_address": false,
            "wait_for_ip_address_timeout": 300
        }
    },
    "msg": "Failed to create a virtual machine : Customization of the guest operating system 'debian8_64Guest' is not supported in this configuration. Microsoft Vista (TM) and Linux guests with Logical Volume Manager are supported only for recent ESX host and VMware Tools versions. Refer to vCenter documentation for supported configurations."  
}
```

`clone_customize_guest_test.yml` works fine with the same configuration.

Reply:
That's weird. Was the test vm not turned off when you ran the test?